### PR TITLE
rg-168: Wire up resume page

### PR DESCRIPTION
**Pull request by Icerock10**
https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-resumegemm/assets/24661681/09af7eaa-96d9-4e27-859c-85ec2bbe8935

https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-resumegemm/assets/24661681/44d0756f-5ae4-4eda-89d9-61179f2ba399

- create resume
- edit resume
- request ai review
- generate pdf
- share resume link

What is missing:

- take a screenshot of the resume
- appropriate template settings shape with id which corresponds to the fields in DB

### CODE REVIEW

**Review by MaxPoplavskiy — APPROVED** [review]

**Comment by vlad-blip** [comment]
[Wire up Resume page](https://trello.com/c/ClyDlIbD/168-wire-up-resume-page)

File: frontend/src/bundles/resume/pages/create-resume/create-resume.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        <>
+            <Header />
+            <div className={styles.create__resume}>
+                {templateSettings && (
+                    <>
+                        <OnlineEditorTabsHandler
+                            tabs={templateSettings.containers}
+                            isCreate
+                        />
+                        <TemplateEditor
```
Comment (honkster-nazarii) [inline frontend/src/bundles/resume/pages/create-resume/create-resume.tsx:23]: consider creating a separate component for preview so it doenst have drag and drop functions

File: frontend/src/bundles/resume/pages/edit-resume/edit-resume.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+const EditResume: React.FC = () => {
+    const { templateSettings, setTemplateSettingsMockData } = useResumes();
+    return (
+        <div className={styles.edit__resume}>
+            {templateSettings && (
+                <>
+                    <OnlineEditorTabsHandler
+                        tabs={templateSettings.containers}
+                    />
+                    <TemplateEditor
```
Comment (honkster-nazarii) [inline frontend/src/bundles/resume/pages/edit-resume/edit-resume.tsx:16]: check prev comment

File: frontend/src/bundles/resume/pages/create-resume/create-resume.tsx (hunk 1 above)
Comment (Icerock10) [reply frontend/src/bundles/resume/pages/create-resume/create-resume.tsx:23]: Yeah, good notice, was going to do so

File: frontend/src/bundles/resume/components/resume-editor/resume-editor-container/resume-editor-container.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { type LayoutContainer } from '~/bundles/resume/types/types';
+
+import { ResumeEditorBlock } from '../resume-editor-block/resume-editor.block';
+
+const ResumeEditorContainer: React.FC<LayoutContainer> = ({
```
Comment (honkster-nazarii) [inline frontend/src/bundles/resume/components/resume-editor/resume-editor-container/resume-editor-container.tsx:5]: I think I made these components already) check RegularItem/Block/Container

Comment (Icerock10) [reply frontend/src/bundles/resume/components/resume-editor/resume-editor-container/resume-editor-container.tsx:4]: y, regular block could be reused)

**Review by honkster-nazarii — APPROVED** [review]

**Review by MaxPoplavskiy — CHANGES_REQUESTED** [review]

**Review by MaxPoplavskiy — APPROVED** [review]

**Pull request merged**